app/workers/moderator/unpublish_all_articles_worker.rb

```diff
@@ -22,6 +25,9 @@ def clean_up_notifications(article)
       Notification.remove_all_by_action_without_delay(notifiable_ids: article.id,
                                                       notifiable_type: "Article",
                                                       action: "Published")
+
+      ContextNotification.delete_by(context_id: article.id, context_type: "Article", action: "Published")
+
```

Comment:
Was this accidentally commited? it's part of another PR.

Reply:
Actually, I based this branch on that one :see_no_evil: 
I will mark as WIP for now, and then remove when that one is merged.